gemfirexd/core/src/main/java/com/pivotal/gemfirexd/internal/iapi/services/context/ContextManager.java

```diff
@@ -716,7 +716,13 @@ else if (finder != null) {
 	 */
 	private void flushErrorString()
 	{
-		errorStream.print(errorStringBuilder.get().toString());
+		//changes to fix SNAP-3176
+		String stmtText = errorStringBuilder.get().toString();
+		if (!stmtText.contains("SET @@session.sql_mode=ANSI_QUOTES") &&
```

Comment:
Do we need a null check for `stmtText` here? Please add if not sure.

Reply:
No, don't need to check